eslint/babel-eslint-parser/test/index.js

```diff
@@ -569,7 +569,7 @@ describe("Babel and Espree", () => {
 
     it("Dynamic Import", () => {
       parseAndAssertSame(`
-        const a = import('a');
+        const a = import(moduleName);
```

Comment:
The test example is revised so that if we do not update `VISITOR_KEYS`, the identifier will not be traversed and the test will fail due to mismatch AST. Note that when traversing AST we remove `loc.identifierName`.